tempesta_fw/http.c

```diff
@@ -748,9 +748,15 @@ tfw_http_adjust_resp(TfwHttpResp *resp, TfwHttpReq *req)
 		return r;
 
 	r = tfw_http_add_hdr_via(hm);
-	if (r)
-		return r;
+	if (resp->flags & TFW_HTTP_RESP_STALE) {
+#define s_110_n "Warning:"
+#define s_110_v "110 - Response is stale"
```

Comment:
Please `#undef` the definitions when they're not needed anymore.
Also, constant definitions must be in upper case according to the coding style.

Also, this is not a requirement, but I would choose a better and meaningful name. This is a `Warning` header field, but the name doesn't tell anything about that. How about `S_WARN_110_N`?